components/src/core/components/InputField/input-group.scss

```diff
@@ -34,3 +34,11 @@
   margin-left: 2px;
   word-break: break-all;
 }
+
+.oxd-input-group__label-icon {
+  width: $oxd-input-group-label-icon-size;
+  height: $oxd-input-group-label-icon-size;
+}
+.oxd-input-group__label-icon.oxd-svg-icon {
+  align-self: end;
```

Comment:
The ._oxd-input-group__label-wrapper_ _align-items_ is set to _baseline_, which is ok for icon font, but does not work for svg icon.